packages/css/src/types.ts

```diff
@@ -452,10 +452,24 @@ export type ThemeUIStyleObject = ThemeUICSSObject | ThemeDerivedStyles
  * An array or object (possibly nested) of related CSS properties
  * @see https://theme-ui.com/theme-spec#theme-scales
  */
-export type Scale<T> = T[] | { [K: string]: T | Scale<T>; [I: number]: T }
+export type Scale<T> =
+  | T[]
+  | {
+    [K: string]: T | null | undefined | Scale<T | null | undefined>;
+    [I: number]: T | null | undefined
+  }
 
 export type TLengthStyledSystem = string | 0 | number
 
+
+export declare namespace ColorMode {
+  // We won't autocomplete "mediumseagreen" inside of `sx` prop, because
+  // csstype's Property.Color is a huge union of literals you didn't include
+  // in your theme. That would be a tiny sabotage, right?
+  // Nevertheless, it's convenient to have them inside of the colors scale.
+  export type ColorValue = CSS.Property.Color;
```

Comment:
@hasparus interesting. Does this mean it doesn't autocomplete but also doesn't break? I might be missing what this does if it can't autocomplete.

Reply:
It autocompletes builtins only inside `theme.colors`, because maybe you want `primary: hotpink` or `primary: ButtonFace`. 

The types for values assignable to `sx.color` and `sx.backgroundColor` is `Scales.Color` (the naming isn't perfect here mby ThemeUIProperty.Color? I'm open for suggestions.)

I aliased it because I was initially adding null here, but I left it to have a place for a comment.